recipe/meta.yaml

```
@@ -14,18 +14,18 @@ build:
 
 requirements:
   run:
-    - cuda-cudart 12.1.105
-    - cuda-nvrtc 12.1.105
-    - cuda-opencl 12.1.105  # [linux64 or win]
-    - libcublas 12.1.3.1
-    - libcufft 11.0.2.54 
-    - libcufile 1.6.1.9  # [linux64]
-    - libcurand 10.3.2.106
-    - libcusolver 11.4.5.107
-    - libcusparse 12.1.0.106
-    - libnpp 12.1.0.40
-    - libnvjitlink 12.1.105
-    - libnvjpeg 12.2.0.2
+    - cuda-cudart 12.2.140
+    - cuda-nvrtc 12.2.140
+    - cuda-opencl 12.2.140  # [linux64 or win]
+    - libcublas 12.2.5.6
+    - libcufft 11.0.8.103 
+    - libcufile 1.7.2.10  # [linux64]
```

Should this be updated to include `linux_aarch64`?

```suggestion
    - libcufile-dev 1.7.2.10    # [linux64 or aarch64]
```